plugins/@grouparoo/salesforce/src/lib/export/model.ts

```diff
@@ -32,8 +32,7 @@ export function getSalesforceModel(
 
   const modelKeys = Object.keys(model);
   const refKeys = modelKeys.filter((k) => k.indexOf("Reference") > 0);
-  // refKeys are optional
-  const requiredKeys = modelKeys.filter((k) => !refKeys.includes(k));
+  const requiredKeys = ["recordObject", "recordMatchField"];
```

Comment:
There is something to do in this file like we did with references where it says `// needs either zero or all refKeys`
And test(s)  to that effect as well.
So if they have 1 of the group ones, they need to have them all.